src/version.json

```diff
@@ -2,12 +2,17 @@
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
   "version": "1.1",
   "assemblyVersion": {
-      "precision": "revision"
+    "precision": "revision"
   },
   "publicReleaseRefSpec": [
     "^refs/heads/main$",
     "^refs/heads/release/v?\\d+(?:\\.\\d+)?$"
   ],
+  "pathFilters": [
+    ".",
+    ":!SDK.sln",
+    ":!PluginsSystem"
```

Comment:
We exclude `PluginsSystem` now so that version.json won't need to change when the plugins stuff gets checked in. Otherwise, SDK version will be bumped.